L/libsingular_julia/common.jl

```diff
@@ -52,11 +49,14 @@ const products = [
 
 # Dependencies that must be installed before this package can be built
 const dependencies = [
-    Dependency("CompilerSupportLibraries_jll"),
-    BuildDependency(PackageSpec(name="Julia_jll", version="v1.4.1")),
+    BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
     Dependency("libcxxwrap_julia_jll"),
     Dependency("Singular_jll"),
```

Comment:
TODO: add a VersionSpec, something like this (depending on whatever version for Singular_jll we end up with)
```suggestion
    Dependency(PackageSpec(name="Singular_jll", version=VersionSpec("400.100.400-400.100"))),
```